bindata/v3.11.0/openshift-apiserver/deploy.yaml

```diff
@@ -68,6 +57,9 @@ spec:
               echo "Copying system trust bundle"
               cp -f /var/run/configmaps/trusted-ca-bundle/tls-ca-bundle.pem /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
             fi
+            echo -b "Rotating audit logs in /var/log/openshift-apiserver"
+            test -f /var/log/openshift-apiserver/audit.log && mv /var/log/openshift-apiserver/audit{,-$(date '+%Y-%m-%dT%H-%M-%S.%3N')}.log
```

Comment:
how will it work in a single node env?

pod1 writes to  `/var/log/openshift-apiserver/audit.log`
pod2 comes along and renames it to `/var/log/openshift-apiserver/audit-XYZ.log`

what will `pod1` do if there is no file?
what will `pod1` do when `pod2` creates the file?